app/components/wallet-overview/wallet-overview-redesigned.tsx

```diff
@@ -0,0 +1,211 @@
+
+    usdInDisplayCurrencyFormatted = moneyAmountToDisplayCurrencyString(usdWalletBalance)
+
+    btcInUnderlyingCurrency = formatMoneyAmount({ moneyAmount: btcWalletBalance })
+
+    if (displayCurrency !== WalletCurrency.Usd) {
+      usdInUnderlyingCurrency = formatMoneyAmount({ moneyAmount: usdWalletBalance })
+    }
+  }
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.displayTextView}>
+        <Text type="p1" bold>
+          My Accounts
+        </Text>
+        <GaloyIconButton
+          name={visible ? "eye" : "eye-slash"}
+          size="medium"
+          onPress={() => setVisible(!visible)}
+        />
+      </View>
+      <View style={styles.separator}></View>
+      <View style={styles.displayTextView}>
+        <View style={styles.currency}>
+          <GaloyCurrencyBubble currency="BTC" size={"medium"} />
+          <Text type="p1">Bitcoin</Text>
+        </View>
+        {loading ? (
+          <Loader styles={styles.loaderContainer} />
+        ) : (
+          <HidableArea hidden={visible} style={styles.hiddableArea}>
+            <Text type="p1" bold>
+              {btcInUnderlyingCurrency}
+            </Text>
+            <Text type="p3">{`~ ${btcInDisplayCurrencyFormatted}`}</Text>
+          </HidableArea>
+        )}
+      </View>
+      <View style={styles.separator}></View>
+      <View style={styles.displayTextView}>
+        <View style={styles.currency}>
+          <GaloyCurrencyBubble currency="USD" size={"medium"} />
+          <Text type="p1">Stablesats</Text>
+          <Pressable onPress={() => setIsStablesatModalVisible(true)}>
+            <GaloyIcon
+              color={theme.colors.primary3}
+              backgroundColor={theme.colors.primary9}
+              name="question"
+              size={15}
+            />
+          </Pressable>
+        </View>
+        {loading ? (
+          <Loader styles={styles.loaderContainer} />
+        ) : (
+          <HidableArea hidden={visible} style={styles.hiddableArea}>
+            {usdInUnderlyingCurrency ? (
+              <Text type="p1" bold>
+                {usdInUnderlyingCurrency}
+              </Text>
+            ) : null}
+            <Text type={usdInUnderlyingCurrency ? "p3" : "p1"}>
+              {usdInDisplayCurrencyFormatted}
+            </Text>
+          </HidableArea>
+        )}
+      </View>
+    </View>
+  )
+}
+
+export default WalletOverviewRedesigned
+
+const useStyles = makeStyles(({ colors, mode }) => ({
+  container: {
+    backgroundColor: colors.whiteOrDarkGrey,
+    display: "flex",
+    flexDirection: "column",
+    marginHorizontal: 30,
+    borderRadius: 12,
+    padding: 15,
+  },
+  displayTextView: {
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "space-between",
+    marginVertical: 5,
+  },
+  separator: {
+    height: 1,
+    backgroundColor: mode === "dark" ? colors.lighterGreyOrBlack : colors.primary9,
```

Comment:
this is quite hacky. ideally we should have just one color and its color automatically switch between modes. 

I know it's a pattern that result from our current state of redesign. we can address this in another PR later.